primo-explore/custom/01MIT_INST-TACOS/js/custom.js

```diff
@@ -3,9 +3,37 @@
   var alertBannerTemplate =
     '<div class="full-width-alert">How are we doing? <a href="https://mit.co1.qualtrics.com/jfe/form/SV_0HZvFmPYRjCzSCO">Give us feedback on your search experience</a>.</div>';
 
-  angular
-    .module("viewCustom", ["angularLoad"])
-    .component("prmSearchBookmarkFilterAfter", {
+  var app = angular.module("viewCustom", ["angularLoad"]);
+  app.factory("tacosService", ["$http", function ($http) {
+    return {
+      sendQueryToTacos: function (searchTerm) {
+        var graphQlQuery = `{
+          logSearchEvent(searchTerm: "${searchTerm}", sourceSystem: "primo-testing") {

```

Comment:
During evaluation, and while we're pointing at the staging TACOS instance, this is fine - but obviously when this is in production the value should be adjusted accordingly.

Reply:
what are the possible values here?

Reply:
Sorry, yes - I was looking at `sourceSystem` specifically. I don't know that we've talked about a naming convention specifically for TACOS (maybe @JPrevost remembers but I don't). Bento uses `bento`, and it looks like you're setting up here to use a value like `primo`, which seems like it should be fine.